src/components/Sidebar/Sidebar.module.css

```diff
@@ -0,0 +1,86 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+:root {
+    --main-bg: #27293d;
+    --white: #fff;
+    --grey: #7e879e;
+    --blue: #1d8cf8;
+}
```

Comment:
I merged Miljana's PR please update this PR with newest changes and use the colors she defined